project/controller/massRegistrationController.php

```diff
@@ -33,6 +36,48 @@
 
 
 // methods
+function createMassRegisterResultsTable($arrayOfUserData) {
```

Comment:
We will have many more tables, I would propose to create some utility functions. We would just send data, and table will be echo'ed created automatically.

Something like echoTable($header,$data) and it will iterate through it and print it row by row.

What do you think?

Reply:
That sounds reasonable, Ill make an issue about it and look into it so its done as soon as possible